builds/misc/images-release.yaml

```diff
@@ -326,6 +326,31 @@ jobs:
         inputs:
           PathtoPublish: '$(Build.BinariesDirectory)/publish'
           ArtifactName: 'publish-linux'
+################################################################################
+  - job: linux_API_proxy_module
+################################################################################
+    timeoutInMinutes: 80
+    displayName: linuxAPIProxy
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+      - bash: |
+            docker login '$(build.registry.address)' --username '$(build.registry.user)' --password '$(build.registry.password)'
+            docker login '$(registry.address)' --username '$(registry.user)' --password '$(registry.password)'
+        displayName: 'Docker Login'
+      # Build API Proxy executable
+      - template: templates/build-api-proxy.yaml
+      # Build API Proxy Image
+      - template: templates/image-linux.yaml
+        parameters:
+          name: API Proxy
+          imageName: api-proxy-module
+          project: api-proxy-module      
+      # Check API proxy
+      - task: ComponentGovernanceComponentDetection@0
```

Comment:
I found you tested image release yaml pipeline, https://msazure.visualstudio.com/One/_build/results?buildId=34792030&view=logs&j=5e897c14-3122-5b03-9993-10a307d9da6f&t=5d0133da-3431-5b35-9028-62db81a60a6a.  And there are some security alerts there, do we need to address them?

Reply:
I discussed that with Varun, I think we are. But those keeps bubbling up, so not right now.

Reply:
Emailed to Varun and Hugues will follow up if those images require to address security alerts.